src/lib/heliaFetch.ts

```diff
@@ -126,7 +129,12 @@ function changeCssFontPath (path: string): string {
  *
  */
 export async function heliaFetch ({ path, helia, signal, headers, id, protocol }: HeliaFetchOptions): Promise<Response> {
```

Comment:
we should remove `helia` here and the `get-helia` since we can't pass both custom dnsResolvers and a custom helia instance.